### DISCUSSION TITLE
Welcome to Xoptfoil2 Discussions!

### DISCUSSION THREAD

**Discussion by jxjo** [jxjo/Xoptfoil2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by goney3** [comment]
**Introduction:** I am an ultralight airplane design enthusiast (USA FAR 103 Ultralights). I volunteer with the Ultralight Airplane Workshop on YouTube/Discord and try to work towards a safer airfoil for all, mostly concentrated on safe stall characteristics and low Cm. The ultralight community is rather under-served with limited airfoil choices at slow speeds, and most airfoils are geared towards faster general aviation and much higher Re. I am also an enthusiast of Paraglider design and I am currently working towards constructing a ground handling wing using an airfoil design I worked on with the original xoptfoil by Dan Prosser. Pere Casellas at the Laboratori d'envol has some significant designs as well as open source design software that also uses Fortran for wing design. I am really looking forward to Xoptfoil2 and seeing where it progresses. Thank you for continuing its development!

**Questions:** 
- Does Xoptfoil2 allow for creating closed trailing edge airfoils? I ask as this is a requirement in DIY paraglider designs (the trailing edge is sewn closed).
- Will Xoptfoil2 be able to handle "single skin" airfoil designs? (leading edge and top surface without a bottom). This is an up-and-coming area of DIY paragliding that allows for lighter wings and less packing volume to create wings that can fit in extremely small spaces (great for traveling!).
- Will Xoptfoil2 have the ability to optimize airfoils on a "mush stall characteristic" as an option? I know the last version was more focused on Cl, Cd, and Cm. Our current design does about Cl-max 1.8 with compromise on Cd and Cm to have a jack-of-all-trades airfoil for USA Ultralights. We're looking forward to creating new freely-available airfoils to better serve the average DIY pilot with higher safety.

**Reply by jxjo** [reply]
Hello Keith,
(you replied to the intro discussion topic ;-) may you can create a new discussion topic) 

... this really cool work you are doing. I would be happy to support.

Regarding to your questions: As the aerodynamic engine deep down in the cellar of Xoptfoil is Xfoil, all possibilities and limitations of Xfoil also apply to Xoptfoil2. So maybe you just test the limits ("single skin") with Xflr5? Xfoil is by design not able to calculate real cl ( or cd) near stall and beyond. So I fear for stall questions Xoptfoil2 (Xfoil) is not the right tool.

Best regards 

Jochen
